test/kubemark/start-kubemark.sh

```diff
@@ -504,13 +545,19 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
 
   for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
   do
-    export KUBEMARK_CLUSTER_KUBECONFIG="${RP_KUBECONFIG}-${rp_num}"
-    export RESOURCE_PARTITION_SEQUENCE=${rp_num}
-    create-kubemark-master
+    create_RP ${rp_num} &
```

Comment:
same here for the log mix-up in RP setup routines

Reply:
will add TODO line here too